sample/development/components/integration/test/integrationTests.brs

```diff
@@ -470,6 +482,305 @@ function TS_SDK_integration() as object
             return validator
         end function,
 
+        TC_SDK_createMediaSession: function() as dynamic
```

Comment:
Can we add integration tests for media in a separate file?

Reply:
It's a good suggestion, but the integration test framework does not support it now. Let's update the framework to support it later.